projects/client/src/lib/components/lists/section-list/SectionList.svelte

```diff
@@ -19,6 +19,7 @@
     title,
     item,
     empty,
+    dynamicActions,
```

Comment:
I'll leave it as is for now but ideally we have only an action input and somehow extend rather than having 2 different inputs.

Make a Todo for this item so we revisit later.